fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -90,8 +90,7 @@ def self.wwdr_certificates_installed?
     end
 
     def self.install_wwdr_certificates
```

Comment:
since this only installs one now, maybe we should rename the method
```suggestion
    def self.install_wwdr_certificate
```

Reply:
Ok, agree